app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -288,7 +296,29 @@ protected void load() {
 
     private void loadUrl() {
         setWebSettings(true);
-        mWebView.reloadUrl(mItem.getUrl());
+        reloadUrl(mItem.getUrl());
+    }
+
+    private void reloadUrl(String url) {
+        reloadUrl(url, null);
+    }
+
+    private void reloadUrl(String url, File pdfFile) {
```

Comment:
Can you annotate `pdfFile` argument with `@Nullable`: `@Nullable File pdfFile`? Also I think we need to skip some of logic below if `pdfFile` is null right?

Reply:
Yup